spec/lib/fetch_book_data_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+require 'vcr_helper'
+
+describe FetchBookData, type: :lib, vcr: VCR_OPTS do
+  describe '#subjects' do
+    subject { described_class.new.subjects }
+
+    context 'success' do
+      let(:book_subjects) do
+        ["Math", "Science", "Humanities", "Social Sciences", "Business", "Essentials", "College Success", "High School"]
+      end
+
+      example do
+        expect(subject).to match(book_subjects)
+      end
+    end
+
+    context 'when there is a timeout' do
+      before do
+        # cannot really test read timeout b/c of how webmock inserts itself
+        # https://github.com/bblimke/webmock/issues/286#issuecomment-19457387
+        allow(Net::HTTP).to receive(:start).and_raise(Net::ReadTimeout)
+      end
+
+      it 'returns empty array' do
+        expect(subject).to match([])
+      end
+
+      it 'calls Raven' do
```

Comment:
this could be a tiny bit better if it said it 'calls Raven with a timed out message'.  that is even clearer.

Reply:
Agree; fixed.